src/vaadin-grid-selection-column.html

```diff
@@ -12,12 +12,10 @@
   <template>
     <template class="header" id="defaultHeaderTemplate">
       <vaadin-checkbox class="vaadin-grid-select-all-checkbox" aria-label="Select All" hidden$="[[_selectAllHidden]]" on-checked-changed="_onSelectAllCheckedChanged"
-        checked="[[_isChecked(selectAll, _indeterminate)]]" indeterminate="[[_indeterminate]]">
-      </vaadin-checkbox>
+        checked="[[_isChecked(selectAll, _indeterminate)]]" indeterminate="[[_indeterminate]]"></vaadin-checkbox>
     </template>
     <template id="defaultBodyTemplate">
-      <vaadin-checkbox aria-label="Select Row" checked="{{selected}}">
-      </vaadin-checkbox>
+      <vaadin-checkbox aria-label="Select Row" checked="{{selected}}"></vaadin-checkbox>
```

Comment:
`vaadin-checkbox` `2.2.3` has been released, no need in this change anymore.

Reply:
Grid still depends on `^2.2.1` so in theory this should work with version `2.2.1` also. Imo the change is fine.